File: data/users/admin/PermMissingElem/sharp/done_code.cs
using System;
// you can also use other imports, for example:
// using System.Collections.Generic;

// you can write to stdout for debugging purposes, e.g.
// Console.WriteLine("this is a debug message");

class Solution {
	public int solution(int[] A) {
		// write your code in C# 6.0 with .NET 4.5 (Mono)
      double sum=0;
      for(inti=0;i<A.length;i++)
        sum+=A[i];
      doubleres=0.5*(A.length+1)*(A.length+2)-sum;
      return(int)res;
	}
}

File: data/users/admin/PermMissingElem/c/done_code.c
// you can write to stdout for debugging purposes, e.g.
// printf("this is a debug message\n");

int solution(int A[], int N) {
        // write your code in C99 (gcc 6.2.0)
        int i;
        int tmp_count = 0;
        int tmp_zero  = -1;
        int B[1000000] = {0};

        for (i = 0; i < N; ++i) 
                B[A[i]] = 1;

        for (i = 1; i < 1000000; ++i) {
                if (tmp_count == N) {
                    if (tmp_zero == -1)
                        tmp_zero = N + 1;
                    return tmp_zero;
                }
       
                if (B[i] == 1)
                        tmp_count++;
                
                if (B[i] == 0)
                        tmp_zero = i;
        }
        return tmp_zero;
}
                            
                        